src/main/cli/settings.c

```diff
@@ -750,6 +750,9 @@ const clivalue_t valueTable[] = {
     { "rx_min_usec",                VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { PWM_PULSE_MIN, PWM_PULSE_MAX }, PG_RX_CONFIG, offsetof(rxConfig_t, rx_min_usec) },
     { "rx_max_usec",                VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { PWM_PULSE_MIN, PWM_PULSE_MAX }, PG_RX_CONFIG, offsetof(rxConfig_t, rx_max_usec) },
     { "serialrx_halfduplex",        VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_RX_CONFIG, offsetof(rxConfig_t, halfDuplex) },
+#if defined(USE_RX_MSP_OVERRIDE)
+    { "msp_override_channels",      VAR_UINT32 | MASTER_VALUE, .config.minmaxUnsigned = {0, UINT32_MAX}, PG_RX_CONFIG, offsetof(rxConfig_t, msp_override_channels)},
```

Comment:
Actually, thinking of this, this range should probably be bounded by `{0, (1 << MAX_SUPPORTED_RC_CHANNEL_COUNT) - 1}`.

Reply:
Agreed

Reply:
@mikeller what about `msp_override_channels_mask`

Reply:
Yeap, even better.